CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## 1.17.3-dev
+* Enhancement - Distinct startContainerTimouts for windows/linux
```

Comment:
You should also state that a new config variable for this has been introduced as well

Reply:
OK, sure, I will update both the change log and README file